src/ui/public/vislib/visualizations/gauge_chart.js

```diff
@@ -31,27 +31,48 @@ export function GaugeChartProvider(Private) {
       return function (selection) {
         selection.each(function (data) {
           const div = d3.select(this);
-          const containerMargin = 5;
+          const containerMargin = 20;
           const containerWidth = $(this).width() - containerMargin;
           const containerHeight = $(this).height() - containerMargin;
           const width = Math.floor(verticalSplit ? $(this).width() : containerWidth / data.series.length);
           const height = Math.floor((verticalSplit ? containerHeight / data.series.length : $(this).height()) - 25);
-          const transformX = width / 2;
-          const transformY = self.gaugeConfig.gaugeType === 'Meter' ? height / 1.5 : height / 2;
-
 
+          div
+            .style('text-align', 'center')
+            .style('overflow-y', 'auto');
 
           data.series.forEach(series => {
             const svg = div.append('svg')
-              .attr('width', width)
-              .attr('height', height)
               .style('display', 'inline-block')
-              .style('overflow', 'hidden');
+              .style('overflow', 'hidden')
+              .attr('width', width);
 
-            const g = svg.append('g')
-              .attr('transform', `translate(${transformX}, ${transformY})`);
+            const g = svg.append('g');
 
             const gauges = self.gauge.drawGauge(g, series, width, height);
+
+            if (self.gaugeConfig.type === 'simple') {
+              const bbox = svg.node().children[0].getBBox();
+              const finalWidth = bbox.width + containerMargin * 2;
+              const finalHeight = bbox.height + containerMargin * 2;
+              svg
+                .attr('width', () => {
+                  return finalWidth;
+                })
+                .attr('height', () => {
+                  return finalHeight;
+                });
+
+              const transformX = finalWidth / 2;
+              const transformY = self.gaugeConfig.gaugeType === 'Meter' ? finalHeight / 1.5 : finalHeight / 2;
```

Comment:
This statement does not make sense. I don't think `gaugeConfig.gaugeType` can ever be set to `Meter`. 
* [gaugeConfig.gaugeType](https://github.com/elastic/kibana/blob/master/src/core_plugins/kbn_vislib_vis_types/public/metric.js#L60) can be either 'Arc', 'Circle', 'Metric'
* [gaugeConfig.type](https://github.com/elastic/kibana/blob/master/src/core_plugins/kbn_vislib_vis_types/public/controls/gauge_options.js#L16) can be either 'simple' or 'meter'.